File: Projetos/MVC Pesquisa_Ordenacao/mvc_java/view/Visao.java
package view;

import java.util.List;

/**
 * VIEW: toda a exibição em tela fica aqui.
 */
public class Visao {
    /**
     * método de classe que exibe conteúdo de uma lista de inteiros
     * @param lista
     * @param frase - para exibir no início do método
     */
    public static void exibrLista(List<Integer> lista, String frase) {
        System.out.println(frase);
        for (Object item : lista) {
            System.out.println(item);
        }
        System.out.println("--------------------------");
        System.out.println("Total de registros: " + lista.size());
    }

    /**
     * exibe o tempo gasto por uma rotina
     * @param rotina - número da rotina
     * @param tempoMs - tempo em milissegundos
     */
    public static void exibirTempo(int rotina, long tempoMs) {
        System.out.println("Tempo (ms) rotina " + rotina + ": " + tempoMs);
    }
}

File: Projetos/MVC Pesquisa_Ordenacao/mvc_python/view/visao.py
"""VIEW: toda a exibição em tela fica aqui."""


def exibr_lista(lista, frase):
    """Exibe o conteúdo de uma lista de inteiros."""
    print(frase)
    for item in lista:
        print(item)
    print("--------------------------")
    print("Total de registros:", len(lista))


def exibir_tempo(rotina, tempo_ms):
    """Exibe o tempo gasto por uma rotina."""
    print(f"Tempo (ms) rotina {rotina}: {tempo_ms}")
